TcpLib/TcpHelper.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace TcpLib
+{
+    public static class TcpHelper

```

Comment:
Let's try to avoid using name of helper or utils. Let's be specific about what this class can do. In my opinion, this looks like a TcpMessageHandler